stream_alert_cli/config.py

```diff
@@ -407,18 +412,41 @@ def add_app_integration(self, app_info):
             return
 
         apps_config = cluster_config['modules'].get('stream_alert_apps', {})
-        local_config_keys = {'interval', 'timeout', 'memory'}
         if not exists:
-            # Save a default log level as info to the config
-            app_info['log_level'] = 'info'
-            app_info['current_version'] = '$LATEST'
-            local_config_keys.update({'log_level', 'current_version', 'type'})
-
-            apps_config[app_info['app_name']] = {key: app_info[key] for key in local_config_keys}
+            # Save a default app settings to the config for new apps
+            new_app_config = {
+                'app_name': app_info['app_name'],
+                'concurrency_limit': 2,
+                'current_version': '$LATEST',
+                'handler': 'app_integrations.main.handler',
+                'log_level': 'info',
+                'log_retention_days': 14,
+                'memory': app_info['memory'],
+                'metric_alarms': {
+                    'errors': {
+                        'enabled': True,
+                        'evaluation_periods': 1,
+                        'period_secs': 120
+                    }
+                },
+                'schedule_expression': app_info['interval'],
```

Comment:
Where does the `app_info` object come from? Can we rename `interval` to `schedule_expression` to be consistent?

Reply:
this was actually a bug :) thanks for flagging!